__init__.py

```diff
@@ -27,6 +32,62 @@ def runtime_requirements(self):
                                    no_network_fallback=False,
                                    no_gui_fallback=True)
 
+    def initialize(self):
+        self.precache()
+        self.add_event(f"{self.skill_id}.precache", self.precache)
+        self.bus.emit(Message(f"{self.skill_id}.precache"))
+
+    def precache(self, message: Message = None):
+        """cache soundcloud searches and register some helper OCP keywords
+        populates featured_media
+        """
+
+        def norm(t):
+            return t.split("(")[0].split("[")[0].split("//")[0].replace(",", "-").replace(":", "-").strip()
+
+        artist_names = [v["artist"] for v in self.archive.values()]
+        song_names = [norm(v["title"]) for v in self.archive.values()]
+        playlist_names = [k for k in self.playlists.keys()]
+
+        if message is not None:
+            for query in self.settings.get("featured", ["zz top", "frank sinatra",
```

Comment:
Not a criticism, but wondering how you came up with the default list.  Not bad choices, it gives a variety of music.

Reply:
as skill author i figured its fair if i pick those :D 

its just initial data to ensure something is present in featured media, the goal is that the user sets these according to personal tastes

ofc any suggestions welcome, i tried to not choose black metal :upside_down_face: